index.html

```diff
@@ -0,0 +1,186 @@
+    Fugiat velit maxime mollitia ex molestias totam pariatur quisquam deleniti! Voluptatem reprehenderit tempore illo
+    modi ipsam?
+    Dolore dignissimos tenetur, id iure possimus reiciendis quo ratione quis beatae labore aperiam dicta nihil quas
+    ducimus ab neque. Nesciunt deleniti commodi ut qui corrupti ullam, tenetur modi dicta exercitationem?
+    Magni rem, ad esse culpa deleniti facilis totam necessitatibus laborum, laboriosam et voluptatem quasi? Iusto
+    accusantium hic neque saepe sunt tempora unde libero natus laudantium. Consequatur asperiores assumenda magni
+    accusantium.
+    Fuga reiciendis voluptatibus iste sunt adipisci harum magnam enim repellat quaerat, sapiente, veniam eligendi non
+    facilis sed impedit, aspernatur a. Ea fugit, aliquid cum id nam quaerat eaque. Sint, explicabo!
+    Repellat, praesentium alias cupiditate, magni eveniet voluptates vel inventore sed obcaecati quidem similique
+    voluptatibus quae aperiam impedit illum ullam maxime ipsam sunt molestiae quod. Quo nisi dolorem ratione dolores
+    officia?
+    Nulla quasi voluptatum, earum nostrum inventore molestiae corrupti voluptas id fugit quia nobis commodi dicta? Eius
+    voluptates aliquid ipsum excepturi cumque hic, impedit adipisci numquam. Odio eos facere inventore non.
+    Soluta nostrum voluptates dolore? Nobis, repellat. Aliquam magnam voluptate quo eum consequuntur? Reiciendis unde
+    natus qui laborum expedita repudiandae harum! Cumque dolorem enim molestiae nulla cupiditate! Cupiditate suscipit
+    officia rerum?
+    Itaque voluptate consectetur nulla aliquam minus. Tempora perferendis possimus repellendus ex ipsum eum architecto
+    omnis laudantium voluptatibus! Reiciendis, ea unde perferendis veniam eveniet, iste, neque illo dolore maxime eaque
+    possimus?
+    Distinctio numquam, temporibus provident sunt dignissimos aspernatur eaque nesciunt debitis laborum, quos ad,
+    necessitatibus ipsam molestias voluptatem qui totam ratione veritatis? Explicabo sapiente quis culpa facilis, quos
+    maiores quam maxime.
+    Architecto temporibus ipsa deleniti! Nam magni, odit, natus ad autem numquam, mollitia asperiores nihil dicta
+    molestiae saepe minus. Et porro molestiae quibusdam expedita minima quis suscipit, quae sequi iste deleniti?
+    Dolorem, consectetur, asperiores suscipit magnam rerum adipisci atque ipsa magni voluptatem commodi ullam quibusdam
+    error doloremque ex dignissimos at maxime? Nemo suscipit tempore veniam molestiae, quibusdam accusamus nihil unde
+    adipisci!
+  </div>
+  <!-- <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script> -->
+
+  <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
```

Comment:
- [ ] [Optional] I think it would be better to make a suggestion Instead of giving orders. "_I think it would be better to remove unused comments in your codes to make you code cleaner_" sounds nicer than "_Please remove unused comments in your code, as this makes your code unclean._," doesn't it? :joy: